vdirsyncer/metasync.py

```diff
@@ -55,7 +55,7 @@ async def _resolve_conflict():
         logger.debug(f"B: {b}")
         logger.debug(f"S: {s}")
 
-        if a != s and b != s:
+        if a != s and b != s or storage_a.read_only or storage_b.read_only:
```

Comment:
If either collection is read only, then `_resolve_conflict` will get called, but it doesn't do any special handling for read only collections.

I think this works for you because you likely have a `a wins` (or `b wins`) rule.

I think that, without an `_ wins` this would fail, which sounds sensible in this particular scenario.

Reply:
I have "b wins" rule.